boot.go

```diff
@@ -93,14 +93,14 @@ func (bs *Bootstrapper) Boot() int {
 	logger.Info("Logging initialized")
 
 	health := NewHealth()
+	config := NewLoggingConfig(baseConfig, logger, bs.configMaskedKeys)
```

Comment:
What's a LoggingConfig? It sounds like a config for a logger and not a Config container. Is that what it is, or is it just something that logs the Config values as it injects them into a config?

Reply:
`NewLoggingConfig` is a constructor defined in [go-nacelle/config](https://github.com/go-nacelle/config) that logs the config struct contents as it's loaded.

If that's ambiguous can you open an issue with a suggestion for that particular name?